### ISSUE TITLE
keras and tf.keras has different output

### ISSUE THREAD

**Issue by DLlearn** [tensorflow/tensorflow#35899]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**

- ubuntu 16.04
I create two conda virtualenv.
first:
- tensorflow2.1  gpu  installed by pip
- python 3.6
- cuda 10.1 installed by conda
- cudnn 7.6 installed by conda 
- gpu Nvidia TitanXp with 12GB memory
second:
- tensorflow1.15 cpu 
- keras 2.2.4 installed by conda 
- python 3.6

**Describe the current behavior**

**In the first virtualenv,I used code as below shows:**
`from sklearn.datasets import make_regression
from tensorflow.keras.layers import Dense
from tensorflow.keras.models import Sequential
from tensorflow.keras.optimizers import SGD
from matplotlib import pyplot

X, y = make_regression(n_samples=1000, n_features=20, noise=0.1, random_state=1)

n_train = 500
trainX, testX = X[:n_train, :], X[n_train:, :]
trainy, testy = y[:n_train], y[n_train:]

model = Sequential()
model.add(Dense(25, input_dim=20, activation='relu', kernel_initializer='he_uniform'))
model.add(Dense(1, activation='linear'))

opt = SGD(lr=0.01, momentum=0.9,clipnorm=1)
model.compile(loss='mean_squared_error', optimizer=opt)

history = model.fit(trainX, trainy, validation_data=(testX, testy), epochs=100, verbose=0)

train_mse = model.evaluate(trainX, trainy, verbose=0)
test_mse = model.evaluate(testX, testy, verbose=0)
print('Train: %.3f, Test: %.3f' % (train_mse, test_mse))

pyplot.title('Mean Squared Error')
pyplot.plot(history.history['loss'], label='train')
pyplot.plot(history.history['val_loss'], label='test')
pyplot.legend()
pyplot.show()`
**In the second virtualenv,I just change the code from tensorflow.keras to keras**

**Describe the expected behavior**
`opt = SGD(lr=0.01, momentum=0.9,clipnorm=1)` this code with and without  parameter clipnorm
will have different performance in second virtualenv(keras) but the first virtualenv(tensorflow2.1 tf.keras) just show same output.So is the parameter clipnorm is useless in tensorflow2.1 ?

**Comment by jvishnuvardhan** [comment]
@DLlearn Can you please share a standalone code as a colab gist or jupyter notebook. I tried running your code in `tf-nightly`, and the `loss` defined in the model results `nan`. Please check the [gist here](https://colab.sandbox.google.com/gist/jvishnuvardhan/b2b63e162f03cdf352e45a2c32dc4ac1/untitled762.ipynb). Thanks!

**Comment by DLlearn** [comment]
> @DLlearn Can you please share a standalone code as a colab gist or jupyter notebook. I tried running your code in `tf-nightly`, and the `loss` defined in the model results `nan`. Please check the [gist here](https://colab.sandbox.google.com/gist/jvishnuvardhan/b2b63e162f03cdf352e45a2c32dc4ac1/untitled762.ipynb). Thanks!

@jvishnuvardhan check [pure keras](https://colab.research.google.com/drive/1Xfaxtelvx695b_7JgPcgWoAG4OEHoNZX) colab gist,and [tf keras](https://colab.research.google.com/drive/1fp5VBqLI1kRO2BGSyNEUkoTuuaGSiQDX),compare this two colab gist,you can find the parameter of SGD optimizer in tensorflow.keras is useless, I do not know why tensorflow keep it as a parameter.

**Comment by tanzhenyu** [comment]
I just ran it. Looks like this has been fixed. Please try and verify it in tf-nightly-gpu.
